src/gc/gc.cpp

```diff
@@ -26870,7 +26868,7 @@ BOOL gc_heap::create_bgc_thread_support()
     }
 
     //needs to have room for enough smallest objects fitting on a page
-    parr = new (nothrow) (uint8_t* [1 + page_size / MIN_OBJECT_SIZE]);
+    parr = new (nothrow) uint8_t*[1 + GCToOSInterface::GetPageSize() / MIN_OBJECT_SIZE];
```

Comment:
@Maoni0 This new feeds c_mark_list indirectly.  It was switched from new of a compile time determined type to new [] with a variable sized type.